src/actions/types.ts

```diff
@@ -194,8 +181,8 @@ export type ActionsAuthType = ActionAuthFetchUser;
 export type ActionsAlertsType = ActionPushAlert | ActionPopAlert;
 export type ActionsWaiverType =
     | ActionWaiverCreate
-    | ActionWaiverResetReply
-    | ActionWaiverResult;
+    | ActionWaiverReset
```

Comment:
Matej, let's keep in separated in PR code improvements and actual required change for the package upgrade.
With this for me it is hard to track where it this change improvement or required change for package update.